src/integrations/prefect-aws/prefect_aws/workers/ecs_worker.py

```diff
@@ -261,7 +261,7 @@ class ECSJobConfiguration(BaseJobConfiguration):
     Job configuration for an ECS worker.
     """
 
-    aws_credentials: Optional[AwsCredentials] = Field(default_factory=AwsCredentials)
+    aws_credentials: Optional[AwsCredentials] = Field(None)
```

Comment:
Why did you remove the `default_factory` here?

Reply:
hmm I guess I was thinking the default factory seemed not so helpful, but now im realizing it probably scoops env via boto on the worker?

Reply:
restored in [d25d6c6](https://github.com/PrefectHQ/prefect/pull/13971/commits/d25d6c647a232ed2c3ea50d3ec9fe7754d388e37)